src/vulkan/context.c

```diff
@@ -524,6 +527,9 @@ pl_vk_inst pl_vk_inst_create(pl_log log, const struct pl_vk_inst_params *params)
             .sType = VK_STRUCTURE_TYPE_APPLICATION_INFO,
             .apiVersion = api_ver,
         },
+#ifdef VK_KHR_portability_enumeration
+        .flags = VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR
```

Comment:
Should this flag only be set if the `VK_KHR_portability_enumeration` extension was enabled successfully?